.unreleased/pr_6360

```diff
@@ -0,0 +1,3 @@
+Implements: #6360 Remove old Continuous Aggregate format
```

Comment:
CAggs with old format are still supported afterward. However, no new ones can be created.

```suggestion
Implements: #6360 Remove support for creating Continuous Aggregates with old format 
```

Reply:
Fixed